packages/react-core/src/helpers/resizeObserver.tsx

```diff
@@ -70,7 +70,7 @@ import { canUseDOM } from './util';
 export const getResizeObserver = (
   containerRefElement: Element,
   handleResize: () => void,
-  useRequestAnimationFrame: boolean = true
+  useRequestAnimationFrame?: boolean
```

Comment:
IS it worth adding some comment here about when to set this?

Reply:
It's blocked out in the files tab, but I did previously add this description for the parameter:

```
* @param {boolean} useRequestAnimationFrame Whether to pass the handleResize function as a callback to requestAnimationFrame. Pass in false when the function passed in is debounced. Defaults to true.
```

That said I just pushed another commit to update the description since it was slightly inaccurate after this update.